index.html

```diff
@@ -25,29 +27,92 @@
             <div class="profile">
 
             </div>
-            
+
         </aside>
         <!-- LEFT SIDE end -->
-        
+
         <!-- MAIN SECTION -->
         <main>
-            <div class="tomiris"></div>
-            <div class="second-tomiris"></div>
-        </main>
-        <!-- MAIN SECTION -->
+            <!-- input post -->
+            <div class="post">
+                <input type="text"  value="Status: Feeling Blue"/> 
+                <div class="btn"><i class="fas fa-pencil-alt"></i> Post</div>
+            </div>
 
-        <!-- RIGHT SIDE -->
-        <aside class="right">
-            <p>test</p>
-        </aside>
-        <!-- RIGHT SIDE end -->
+
+            <!-- post a -->
+            <div class="post">
+                <div class="avatar-name">
+                    <div class="avatar a"></div>
+                    <div class="name">Eugene</div>
+                </div>
+                <div class="min">1 min</div>
+                <div class="clr"></div>
+                <hr />
+                <p>Next cohort will start soon. We strive for teaching our students the most up-to-date programming
+                    technologies. Contact us.</p>
+                <div class="img-wrapper">
+                    <img class="img-a" src="https://learn.seytech.co/assets/projects/html-css/projects/6-social-network/assets/p0.png" alt="Seytech Merch">
+                        
+                    <img class="img-a" src="https://learn.seytech.co/assets/projects/html-css/projects/6-social-network/assets/p1.jpg"
+                        alt="coding mind">
+                </div>
+
+
+                <div class="btn"><i class="fas fa-thumbs-up"></i> Like</div>
```

Comment:
buttons should be in a wrapper. If you have something similar more than one, put them in a wrapper - union